moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -692,27 +544,25 @@ const ProjectComponentEdit = ({
     // eslint-disable-next-line
   }, [saveActionState]);
 
-  if (loading) return <CircularProgress />;
-  if (error) return <div>Error: {JSON.stringify(error)}</div>;
-
   /**
    * Returns true if the collection has a minimum of features, false otherwise.
    * @type {boolean}
    */
-  const areMinimumFeaturesSet =
-    countFeatures(editFeatureCollection) >= mapConfig.minimumFeaturesInProject;
+  const areMinimumFeaturesSet = editFeatureCollection
+    ? countFeatures(editFeatureCollection) >= mapConfig.minimumFeaturesInProject
+    : false;
```

Comment:
because `editFeatureCollection` is now hooked, it is `null` on first render—so we gotta handle that.